### ISSUE TITLE
rspec adds unrelated spec files to the backtrace

### ISSUE THREAD

**Issue by carllerche** [rspec/rspec-core#101]
On rspec 2.0.0.beta.19 running the entire spec suite with "rake" seems to add an extra line to the backtrace in errors that appears to be unrelated:

  1) bundle install with git sources switching sources doesn't explode when switching Path to Git sources
      Failure/Error: should_be_installed "foo 1.0", "bar 1.0"
      expected: #<Gem::Version "1.0">,
           got: #<Gem::Version ""> (using ==)
      Diff:
      @@ -1,2 +1,2 @@
      -Gem::Version.new("1.0")
      +Gem::Version.new("")
      # ./spec/support/matchers.rb:38:in `should_be_installed'
      # ./spec/support/matchers.rb:33:in`each'
      # ./spec/support/matchers.rb:33:in `should_be_installed'
      # ./spec/install/git_spec.rb:453
      # ./spec/cache/gems_spec.rb:3

the bottom line doesn't seem to belong.
